### ISSUE TITLE
When does the token expire?

### ISSUE THREAD

**Issue by edent** [edent/Renault-Zoe-API#1]
Seems to have a refresh token, but no indication of when it needs to be used.

Certainly expires after 24 hours.

**Comment by cwimberger** [comment]
I think the best way would be to listen for the ExpiredToken response and refresh the token accoringly.

```
{
  "code": 403852,
  "message": "com.worldline.renault.myzeonline.exception.InvalidAuthenticationException.ExpiredToken",
  "timestamp": 1481443268063
}
```
At the moment, I'm trying to find the service call to refresh it.

**Comment by daniarv** [comment]
Hi,

Have you found any solution to auto refresh the token when it has expired?

I'm trying to show my car status on a local webpage and have the page to refresh every 10 minutes.
Maybe that's the reason my token expire too quickly, I have reached the limit of API calls...

**Comment by matthew1471** [comment]
The token is a JSON Web Token (JWT). You can copy it into https://jwt.io/ and see the expiry date for any given token. I believe the tokens expire after 10 hours.

I have an open pull request (#12) to add token support and fix some minor issues :).

**Comment by edent** [comment]
Closed in #13
